ScreenSelectorExample/WindowsPlayer/WindowsPlayer/Main.cpp

```diff
@@ -0,0 +1,22 @@
+#include "PrecompiledHeader.h"
+#include "..\UnityPlayerStub\Exports.h"
+#include "ScreenSelector.h"
+
+#include <string>
+
+// Hint that the discrete gpu should be enabled on optimus/enduro systems
+// NVIDIA docs: http://developer.download.nvidia.com/devzone/devcenter/gamegraphics/files/OptimusRenderingPolicies.pdf
+// AMD forum post: http://devgurus.amd.com/thread/169965
+extern "C"
+{
+__declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
```

Comment:
Which main.cpp are you talking about? It's already in main.cpp.

On Windows, these symbols have to be exported from the .exe. Exporting it from UnityPlayer.dll doesn't work (I tried it when I split executable and UnityPlayer.dll apart).

Reply:
Well on Linux we have them in main.cpp not LinuxMain.cpp So they are in UnityPlayer.so on Linux.

Reply:
Likely. On Linux, if you do `dlopen(RTLD_DEFAULT, "NvOptimusEnablement")` it will find it no matter which .so file it's part of. On Windows you'd need to enumerate every single loaded DLL and then call GetProcAddress on it. I bet drivers don't do that and instead do `GetProcAddress(GetModuleHandleW(nullptr), "NvOptimusEnablement")` which only searches the .exe.